UserSearcArray.java

```diff
@@ -0,0 +1,76 @@
+import java.util.Scanner;
+
+public class UserSearcArray {
+
+	int[] userarr;
+	String[] emailarr;
+	Scanner sc;
+
+	public void createUseridArray() {
+		userarr = new int[5];
+
+		System.out.println("Enter User id in user id array....");
+		sc = new Scanner(System.in);
+
+		for (int i = 0; i < userarr.length; i++) {
+			System.out.println("Enter User id in user id in " + i + " place...");
+			userarr[i] = sc.nextInt();
+		}
+
+	}
+
+	public void createEmailIdArray() {
+		emailarr = new String[5];
+
+		System.out.println("Enter email id in email array....");
+		sc = new Scanner(System.in);
+
+		for (int i = 0; i < emailarr.length; i++) {
+			System.out.println("Enter email id in " + i + " place...");
+			emailarr[i] = sc.next();
+		}
+
+	}
+
+	public void printUseridWithEmailArray() {
+
+		for (int i = 0; i < userarr.length; i++) {
```

Comment:
Please use appropriate names instead of i or j e.g. userIdIndex, emailIdIndex. Names should be meaningful always